src/main/python/calibration/generate_rides_with_google_maps.py

```diff
@@ -0,0 +1,98 @@
+import pandas as pd
+import sys
+from pathlib import Path
```

Comment:
can we provide url/gzip file and will that work as well?

Reply:
in theory it should. can you provide a link so as to I can test?